src/library/styles/base/_type.scss

```diff
@@ -1,57 +1,3 @@
-body {
-  color: $text-color-primary;
-  @include font-family-base-regular;
-  @include font-size($fs-body);
-  line-height: 1.5;
-}
-
-p,
-dl {
-  margin: space(2x) 0 0;
-}
-
-ul,
-ol {
-  margin: space(2x) 0 0;
-  padding-left: space(4x);
-}
-
-li {
-  &:not(:first-of-type) {
-    margin: space(2x) 0 0;
-  }
-}
-
-figure {
-  position: relative;
-  margin: space(4x) space(4x) 0;
-
-  blockquote {
-    margin: 0;
-  }
-
-  figcaption {
-    margin: space(md) 0 0;
-    padding-left: space(3x);
-
-    &::before {
-      position: absolute;
-      left: 0;
-      content: '—';
-    }
-  }
-}
-
-cite {
-  font-style: normal;
-}
-
-h1,
-h2,
-h3,
-h4,
-h5,
-h6,
 .bsds-h1,
 .bsds-h2,
 .bsds-h3,
```

Comment:
i can't comment on the line, but we'll want to pull margins out of lib and into docs, e.g.: L67